### DISCUSSION TITLE
Updating `launcher-package` project

### DISCUSSION THREAD

**Discussion by ptrdom** [sbt/sbt#7395] [Ideas]
While working on https://github.com/sbt/sbt/issues/7387 I noticed that `launcher-package` project is build with old sbt version - `0.13.18`, also its main integration test is testing an old sbt version - `1.3.13`. Would it be beneficial to spend time updating `launcher-package` project and its dependencies?

**Comment by eed3si9n** [comment]
Yea, I don't remember the exact details why it is the way it is (probably too much Linux details based on sbt-native-packager?), but if you'd like to tackle bumping it up to a modern sbt that'd be awesome.

**Reply by ptrdom** [reply]
Cool, I will give it a shot then.
